homework/index.js

```diff
@@ -30,14 +30,77 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    leftContainer.innerHTML = '';
+    const table = createAndAppend('table', leftContainer);
+    const tBody = createAndAppend('tBody', table);
+    const trRepository = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trRepository, { text: 'Repository: ' });
+    createAndAppend('a', trRepository, {
+      text: repository.name,
+      href: repository.html_url,
+    });
+    const trDescription = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trDescription, { text: 'Description: ' });
+    createAndAppend('td', trDescription, { text: repository.description });
+    const trForks = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trForks, { text: 'Forks: ' });
+    createAndAppend('td', trForks, { text: repository.forks });
+    const trUpdated = createAndAppend('tr', tBody, { class: 'table-info' });
+    createAndAppend('td', trUpdated, { text: 'Updated: ' });
+    createAndAppend('td', trUpdated, { text: repository.updated_at.toLocaleString() });
+  }
```

Comment:
There is some repeating code here: create a `tr`, append a `td` and then another `td`. Do you think you could extract this into a function, perhaps named `addRow`, to eliminate this code duplication?

Reply:
Hello @remarcmij code has been improved based on your comment.
Thanks.